throughput/src/bin/z_sub_thr.rs

```diff
@@ -52,6 +52,10 @@ struct Opt {
     /// declare a numerical Id for the subscribed key expression
     #[clap(long = "declare_expr")]
```

Comment:
Maybe we can directly follow the convention of command-line arguments.

By changing the writing to below,

```rust    
    #[clap(long)]
    use_expr: bool,

    /// do not use callback for subscriber
    #[clap(long)]
    no_callback: bool,
```
we can call the program like `z_sub_thr --no-callback --declare-expr`.

Reply:
Agree. Will update.

Reply:
@william-wyliang  I have a small suggestion. Although you can specify the explicit name of a long argument, the default behavior is to change `use_expr` in Rust to `--use-expr` while using command-line arguments. So I think we can omit the term `#[clap(long = "...")]`. A simple `#[clap(long )]` is enough.

Reply:
Another PR has been submitted for this.